### DISCUSSION TITLE
Why are releases deleted?

### DISCUSSION THREAD

**Discussion by ghost** [Acode-Foundation/Acode#520] [General]
![Screenshot_20220913-223542_Chrome](https://user-images.githubusercontent.com/78432747/190004209-09c44e4f-adc2-42dd-8fb0-5604723fbd40.png)

**Comment by IzumiSenaSora** [comment]
Same question!

**Comment by deadlyjack** [comment]
There were multiple places where I have to upload the same apk. The same build can be downloaded from telegram group.

**Reply by IzumiSenaSora** [reply]
Why not just automate it by github action?
